internal/kibana/reconciler.go

```diff
@@ -577,14 +596,19 @@ func newKibanaPodSpec(cluster *KibanaRequest, elasticsearchName string, proxyCon
 		*kibanaProxyResources,
 	)
 
+	cookieExpire := "24h"
```

Comment:
`cookieExpire` could be of type `time.Duration`, as you are using `fmt.Sprintf` with a `%s` marker anyway, you can avoid the explicit call to `String()` and provide a correctly-typed default value.

Reply:
makes sense